string object_type
 ---
float64 value